crates/typst-layout/src/flow/collect.rs

```diff
@@ -466,6 +467,7 @@ impl<'a> MultiChild<'a> {
         let mut spill = None;
         if frames.next().is_some() {
             spill = Some(MultiSpill {
+                exist_non_empty_frame,
```

Comment:
Minor comment, but it'd be a good idea to inline the variable here, especially since its calculated value is discarded outside of this branch.